src/Platform/Hosted/Components/FeedbackCommandButton.tsx

```diff
@@ -1,22 +1,18 @@
 import * as React from "react";
-import { CommandButtonComponent } from "../../../Explorer/Controls/CommandButton/CommandButtonComponent";
 import FeedbackIcon from "../../../../images/Feedback.svg";
 
 export const FeedbackCommandButton: React.FunctionComponent = () => {
+  const onClick = () => {
+    window.open("https://aka.ms/cosmosdbfeedback?subject=Cosmos%20DB%20Hosted%20Data%20Explorer%20Feedback");
+  };
+
```

Comment:
nit: consider moving the method outside of the component to avoid instantiating the object on every render.